source/clear-linux/guides/tooling/autospec.rst

```diff
@@ -322,6 +322,117 @@ and dependencies in order for autospec to complete a successful build.
    if autospec is not able to automatically find the correct dependency on its
    own.
 
+Test packaged software
+**********************
+
+After software has been packaged with autospec, the resulting RPMs can be
+tested for functionality before being integrated and deployed into a |CL|
+image with the :ref:`Mixer tool <mixer>` or :ref:`Mixin tool <mixin>`.
+
+The |CL| development tooling offers two ways to quickly test autospec
+generated RPMs.
+
+.. note::
+   The methods outlined below should only be used for temporary testing on
+   development systems. 
+
+
+Test in a |CL| virtual machine
+==============================
+
+The |CL| development tooling includes a method to mount RPMs into a |CL|
+virtual machine running on the KVM hypervisor. Using a :abbr:`VM (Virtual
+Machine)` allows testing is completely isolated environment. 
+
+To test an autospec created package inside a VM:
+
+#. Download the |CL| KVM image into the :file:`~/clearlinux` directory as
+   :file:`clear.img`. The location and name :file:`clear.img.xz` is important
+   for the tooling to work:
+
+   .. code-block:: bash
+
+      cd ~/clearlinux
+      curl -o clear.img.xz https://download.clearlinux.org/image/$(curl https://download.clearlinux.org/image/latest-images | grep '[0-9]'-kvm)
+
+#. Extract the downloaded |CL| KVM image:
+
+   .. code-block:: bash
+
+      unxz -v clear.img.xz
+
+#. Copy the QEMU start script and virtual firmware needed for KVM into the
+   :file:`~/clearlinux` directory:
+
+   .. code-block:: bash
+
+      cp ~/clearlinux/projects/common/start_qemu.sh .
+      cp /usr/share/qemu/OVMF.fd .
```

Comment:
`OVMF.fd` lives in projects/common nowadays too, so you can copy it from there.

Reply:
I think there was an effort to standardize documentation on the `/usr/share` path for consistency.

Reply:
Okay, that's fair.